### ISSUE TITLE
Add LICENSE to repository

### ISSUE THREAD

**Issue by TimMurphy** [gregoryyoung/m-r#12]
It would be helpful to explicitly know what the license is for this repository.

For instance - can I fork and refactor for training purposes?

**Comment by beachwalker** [comment]
Good point. Some did already a "fork" of this or at least took some inspiration from here (so to say). Look at [CQRSlite](https://github.com/gautema/CQRSlite) for example.

**Comment by TimMurphy** [comment]
@gregoryyoung
